retis-events/src/python.rs

```diff
@@ -75,9 +75,10 @@ impl PyEvent {
     /// Allows to use the object as a dictionary, eg. `e['skb']`.
     fn __getitem__<'a>(&'a self, py: Python<'a>, attr: &str) -> PyResult<Py<PyAny>> {
         if let Ok(id) = SectionId::from_str(attr) {
-            if let Some(section) = self.0.get(id) {
-                return Ok(section.to_py(py));
-            }
+            return Ok(match self.0.get(id) {
+                Some(section) => section.to_py(py),
+                None => py.None(),
```

Comment:
All the benefits you mention are true. However, this is completely against python "standards". I'm not sure if there is a PEP for this but en every object out there, if it supports `__getitem__` it returns a `KeyError` if the key does not exist. Not throwing but returning `None` would be extremely confusing IMHO.

Given that `__getitem__` has such a well established semantics, I would avoid breaking it here and instead add another helper such as the one in `dict`: https://docs.python.org/3.8/library/stdtypes.html#dict.get

That would allow a cleaner syntax ( `if event.get("skb", None): [...]`) which is both more explicit and does not break existing semantics.